### DISCUSSION TITLE
Server config problem

### DISCUSSION THREAD

**Discussion by TimRuskof** [Vdauphin/HeartsAndMinds#1491] [Question and answer]
Hi ! 
When putting the mission file on the dedicated server of my team, there is no air, and sea units spawning, but there is when I host with my own pc.
Do you know where are the settings about that in the arma 3 server settings ?

**Comment by Vdauphin** [comment]
Hello,
There is no such setting. May be your server is just broken. Check your .rpt and fix all error
Cheers
